sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -462,6 +462,7 @@ package object dsl {
         Window(windowExpressions, partitionSpec, orderSpec, logicalPlan)
 
       def subquery(alias: Symbol): LogicalPlan = SubqueryAlias(alias.name, logicalPlan)
```

Comment:
I don't know why do we need to accept a `Symbol` here. We can probably do a cleanup later and remove this method. The same to `def as(alias: Symbol): NamedExpression = Alias(expr, alias.name)()` in this file.

Reply:
I looked up the history, it is added at the beginning of SQL ..

Reply:
sure, will do a cleanup later